docker/README.md

```diff
@@ -19,3 +19,40 @@ Build the container, for python users:
 After build finished, run the container:
 
     docker run --rm -it lightgbm
+
+## Using CLI via Docker
+
+Build a Docker image with LightGBM CLI:
+
+    docker build -t lightgbm-cli -f dockerfile-cli .
+
+where `lightgbm-cli` is the image name.
```

Comment:
We don't need this line since we have precise wget command above and user shouldn't worry about filename.

Reply:
`lightgbm-cli` doesn't refer to a file name but rather to the resulting Docker image name, the one that the user will have to type in the next command.

Reply:
Sorry, I mixed up `lightgbm-cli` and `dockerfile-cli` (thought the note was about `dockerfile-cli`).